### PR TITLE
PAS-421 | Add community SDKs to the Get Started page

### PR DESCRIPTION
**Pull request by Tyrrrz**
### Ticket

- Closes https://bitwarden.atlassian.net/browse/PAS-421

### Description

This adds to several community SDKs and changes the text a bit. Open for discussion in terms of presentation.

### Screenshots

image

Will update the icons once we agree this is the direction we want to go with the layout.

### Checklist
I did the following to ensure that my changes were tested thoroughly:
- __

I did the following to ensure that my changes do not introduce security vulnerabilities:
- __

### CODE REVIEW

File: src/AdminConsole/Components/Pages/App/Onboarding/GetStarted.razor [hunk 1]
```diff
@@ -55,13 +55,13 @@
             </PanelFooter>
         </Panel>
 
-        <Panel Header="3. Frontend framework">
+        <Panel Header="3. Frontend">
             <div class="mt-2 max-w-xl text-sm text-gray-500">
-                <p>Passwordless.dev uses a little bit of JavaScript and HTML. It works regardless if you're using a framework or not! For some frameworks, we provide official packages, SDKs and examples.</p>
+                <p>Passwordless.dev uses a little bit of JavaScript and HTML. It works regardless if you're using a framework or not! There are also official and community-maintained SDKs available for several platforms.</p>
```
Comment (jonashendrickx) [inline src/AdminConsole/Components/Pages/App/Onboarding/GetStarted.razor:60]: Maybe remove the Javascript sentence altogether.

**Comment by jonashendrickx** [comment]
> I think featuring the community SDK's is fine. Perhaps we could add more distinciton with a "official" / community badge as well to make sure none misses the low-contrast description text?

But we have to remember they may or may not be maintained. And I don't know whether they were all properly tested.

**Comment by Tyrrrz** [comment]
Yeah I think showing them so prominently kind of makes them our responsibility, so if the SDK breaks or becomes unsupported, customers will expect us to help them with it

**Comment by abergs** [comment]
@Tyrrrz Ok - what about changing so that we instead:

Create a documentation page "Community SDK".
On that page, mention that these are SDK's built by the community and not maintain by Bitwarden.

Link to the documentation page in Admin console:

> Looking for Rust, Go or another language? Check out the Community SDKs ->

**Comment by Tyrrrz** [comment]
Superseded by https://github.com/bitwarden/passwordless-docs/pull/117

**Pull request closed without merging**